guides/request-validation-php-lumen/example-2/example-2.5.x.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Http\Request;
+use Twilio\Twiml;
+
+$app->post('/voice', ['middleware' => 'TwilioRequestValidator',
```

Comment:
doesn't looks like it is needed. I modified the code and ran it and it works perfectly without it. thanks!